src/ontology/mondo-ingest.Makefile

```diff
@@ -588,11 +591,52 @@ $(REPORTDIR)/%.subclass.confirmed.robot.tsv $(REPORTDIR)/%.subclass.added.robot.
 ##################################
 ## Externally managed content ####
 ##################################
-# General
-update-externally-managed-content: external-content-nord external-content-nando external-content-ordo external-content-omim
 
 EXTERNAL_CONTENT_DIR=external
 
+EXTERNAL_FILES = \
+	efo-proxy-merges \
+	mondo-clingen \
+	mondo-efo \
+	mondo-medgen \
+	mondo-omim-genes \
+	mondo-otar-subset \
+	nando-mappings \
+	gard \
+	nord \
+	ordo-subsets
+
+# The following target simulates the EMC pipeline. It runs it in exactly the same way
+# as it would be run in the context of the Mondo repo. This way, we can test Mondo 
+# in exactly the same state as it would look like when the EMC pipeline was run properly.
+# In particular, this allows us to use QC tests that are reliant on the various pre- and 
+# postprocessing steps needed, like Animal QC (which relies on the rare subset pipeline,
+# which is SPARQL postprocessing based).
+
+$(TMPDIR)/mondo-with-simulated-emc-run.owl: $(TMPDIR)/mondo_repo_built $(foreach n,$(EXTERNAL_FILES), $(EXTERNAL_CONTENT_DIR)/$(n).robot.owl)
+	mkdir -p $(TMPDIR)/mondo/src/ontology/tmp/external/
+	$(foreach n,$(EXTERNAL_FILES), cp -f $(EXTERNAL_CONTENT_DIR)/$(n).robot.owl $(TMPDIR)/mondo/src/ontology/tmp/external/processed-$(n).robot.owl;) \
+	cd $(TMPDIR)/mondo/src/ontology && \
+	touch mondo-edit.owl && \
+	cp mondo-edit.owl mondo-edit.owl.bak && \
+	cp mondo-edit.obo mondo-edit.obo.bak && \
+	make update-external-content-incl-rare -B MIR=false IMP=false MIR=false DOWNLOAD_EXTERNAL=false &&\
+	make mondo-edit.owl MIR=false IMP=false MIR=false
```

Comment:
where is this goal in the Makefile?

Reply:
Looks like it has already made its way into [`mondo` `main` makefile](https://github.com/monarch-initiative/mondo/blob/master/src/ontology/mondo.Makefile#L316).

Reply:
Yes, all that you see here is in the context of the `mondo`, not `mondo-ingest` pipeline!